pkg/types/ovirt/defaults/platform.go

```diff
@@ -1,11 +1,80 @@
 package defaults
 
 import (
+	"fmt"
+
+	"github.com/ovirt/go-ovirt"
+
+	ovirtinstallconfig "github.com/openshift/installer/pkg/asset/installconfig/ovirt"
```

Comment:
In general, we can't have `pkg/types` depend on `pkg/asset`. Defaults set in this pkg will need to be static.

Reply:
yes, not cloud imports in pkg/types